components/storage/blockdevice/COMPONENT_QSPIF/mbed_lib.json

```diff
@@ -38,6 +38,18 @@
             "QSPI_FREQ": "32000000",
             "QSPI_MIN_READ_SIZE": "4",
             "QSPI_MIN_PROG_SIZE": "4"
+        }, 
+        "CY8CPROTO_062_4343W": {
+            "QSPI_FREQ": "50000000",
+            "QSPI_MIN_PROG_SIZE": "512"
```

Comment:
Please note that SFDP standard requires minimum Program-Granularity  of 1byte. 
The standard and our code fully support 512 Bytes Page Size without the need for this parameter.
The parameter was added for supporting devices that couldn't support the standard, and has an impact on Filesystem heap.
If the flash chips your adding support SFDP standard and so does the Cypress target, then this parameter is not required.

Reply:
removed the prog_size setting as both the PSoC 6 and memory chip are compatible.

Reply:
@morser499 - Thanks for the update! I'm not sure whether the test "features-storage-tests-blockdevice-general_block_device" runs as default - it would be a good test to verify the above.

Reply:
I did run those tests, however, I don't have most of the reports anymore. I can rerun them if you would like.  Attached is a screenshot (it isn't letting me upload the html file) of the last log I ran.

![062_ble_gcc](https://user-images.githubusercontent.com/32693569/56900744-69b3e700-6a4b-11e9-8bd1-8da5c8b12714.png)

Reply:
Most important is running "features-storage-tests-blockdevice-general_block_device" after you've removed the prog_size. And making sure in the test report that it ran on QSPIF blockdevice

Reply:
These tests were run after the removal of prog_size. How do I verify it ran on the QSPIF blockdevice?

Reply:
Just uploaded the details for the mbed-os-features-storage-tests-blockdevice-general_block_device (below) which seems to indicate that it was using QSPIF